setup.py

```diff
@@ -25,6 +25,6 @@
         'share/ssl/temboard_CHANGEME.pem',
         'share/ssl/temboard_ca_certs_CHANGEME.pem',
         'share/temboard.conf.sample',
-        'share/temboard.logrotate'
-        'temboardui/plugins/supervision/sql/supervision.sql',
+        'share/temboard.logrotate',
+        'temboardui/plugins/supervision/sql/supervision.sql'
```

Comment:
Note: it's usually a good idea to leave a coma even at the last line of lists in python. It's not a syntax error and avoids modification on the given line if new items are added.